ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -288,7 +288,7 @@
 			attributes = {
 				BuildIndependentTargetsInParallel = YES;
 				LastSwiftUpdateCheck = 1140;
-				LastUpgradeCheck = 1600;
+				LastUpgradeCheck = 1510;
```

Comment:
I think we shouldn't go backwards in this number, it looks like you're using an older version of xcode (`15.1`).  I'd suggest upgrading to xcode 16, so that we don't get a flapping number here each time one of us builds the app.

Alternatively, you can remove this from the PR, and just avoid checking it into any of your commits.

Reply:
<img width="511" alt="Screenshot 2025-01-24 at 10 39 54 AM" src="https://github.com/user-attachments/assets/079f6057-8305-40b7-acc7-c4b254d758a9" />
hmm dunno where it's coming from

Reply:
Removed the changes manually.

Reply:
I actually saw it too, looks like it happens when you start it up via flutter directly, rather than from within xcode.